# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/kdoctools/hotfix-docbook.patch
# Copyright (C) 2022 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- kdoctools-5.86.0/cmake/FindDocBookXSL.cmake.vanilla	2022-09-16 18:10:25.836000000 +0200
+++ kdoctools-5.86.0/cmake/FindDocBookXSL.cmake	2022-09-16 18:17:55.319000000 +0200
@@ -15,7 +15,7 @@
                       )
 
 set (STYLESHEET_PATH_LIST
-    ${CMAKE_INSTALL_DATAROOTDIR}/xml/docbook/stylesheet/docbook-xsl
+    usr/share/xml/docbook/docbook-xsl
     ${CMAKE_INSTALL_DATAROOTDIR}/xml/docbook/xsl-stylesheets
     ${CMAKE_INSTALL_DATAROOTDIR}/sgml/docbook/xsl-stylesheets
     ${CMAKE_INSTALL_DATAROOTDIR}/xml/docbook/stylesheet/nwalsh/current
--- kdoctools-5.86.0/cmake/FindDocBookXML4.cmake.vanilla	2022-09-16 18:08:05.420000000 +0200
+++ kdoctools-5.86.0/cmake/FindDocBookXML4.cmake	2022-09-16 18:12:57.968000000 +0200
@@ -28,7 +28,7 @@
 function (locate_version version found_dir)
 
     set (DTD_PATH_LIST
-        ${CMAKE_INSTALL_DATAROOTDIR}/xml/docbook/schema/dtd/${version}
+	usr/share/xml/docbook/docbook-xml-dtd-${version}
         ${CMAKE_INSTALL_DATAROOTDIR}/xml/docbook/xml-dtd-${version}
         ${CMAKE_INSTALL_DATAROOTDIR}/sgml/docbook/xml-dtd-${version}
         ${CMAKE_INSTALL_DATAROOTDIR}/xml/docbook/${version}
